hpat/tests/test_series.py

```diff
@@ -67,9 +68,19 @@ class TestSeries(unittest.TestCase):
 
     def test_create1(self):
         def test_impl():
+            # a = 2
```

Comment:
```suggestion
```

Reply:
@Hardcode84 
There is code suggestion tool in GitHub (ctrl+g)

Reply:
Removed